File: test_function.js
exports.handler = async (event, context) => {
    console.log('Event:', JSON.stringify(event, null, 2));
    console.log('Context:', JSON.stringify(context, null, 2));
    
    return {
        statusCode: 200,
        body: JSON.stringify({
            message: 'Hello from Lambda@Home!',
            event: event,
            timestamp: new Date().toISOString()
        })
    };
};

File: test_function.py
def handler(event, context):
    print(f'Event: {event}')
    print(f'Context: {context}')
    
    return {
        'statusCode': 200,
        'body': {
            'message': 'Hello from Lambda@Home Python!',
            'event': event,
            'timestamp': context.get('timestamp', 'unknown')
        }
    }
